core/domain/acl_decorators.py

```diff
@@ -482,6 +482,10 @@ def test_can_translate(self, exploration_id, **kwargs):
 
         Args:
             exploration_id: str. The exploration id.
+            **kwargs: dict(str: *). Keyword arguments.
+
+        Returns:
+            Return value of decorated function.
```

Comment:
Suggest:

```
Returns:
    *. The return value of the decorated function.
```

That said -- @apb7 can you fix this and all the docstrings in this file as well? I just realized they seem to be completely wrong -- they talk about returning booleans but as far as I can see these are returning arbitrary values based on the values of the handlers. There is also insufficient documentation about what exceptions are raised and when (there should be a Raises section).

Reply:
Merging this PR in the meantime so that @brianrodri can make progress.

Reply:
Okay, @seanlip. I will fix this soon.

Reply:
Hi @apb7 -- I want to follow up on this. Did you fix this?

Reply:
Hi Sean, I had added this file to #4374 and had mentioned what needs to be done here so as to keep a track of this. I think someone is assigned to this at the moment although it seems that there has been no response from the contributor assigned. Shall I pick this up then? Thanks!

Reply:
Ah ok, thanks! Yes, if you can pick this up, that would be great, because in the meantime the docstrings are actively misleading.